docs/explanation/juju-security.md

```diff
@@ -1,90 +1,732 @@
+- {ref}`threats-integrity`: An attacker with access to an administrator's account might make unauthorized changes to the Juju configuration, such as altering model settings, modifying charm configurations, or changing user permissions.
+
+Controls: {ref}`controls-user-authentication`, {ref}`controls-no-plaintext-passwords-in-the-database`
+
+#### Juju user
+
+A user with access less than controller `superuser` access.
+
+Owned by: Controller.
+
+Used by: Client.
+
+Privileges: As permitted by their access level.
+
+Example threats:
+
+- {ref}`threats-confidentiality`: A Juju user might inadvertently or maliciously expose sensitive data within the Juju model or applications. This could occur if the user accesses sensitive configuration details, logs, or secrets and then shares this information in an unsecured manner, such as storing it in an unprotected location or sending it over an insecure channel.
+
+Controls: {ref}`controls-user-authentication`,  {ref}`controls-granular-access`, {ref}`controls-no-plaintext-passwords-in-the-database`
+
+(assets-workloads)=
+### Workloads
+
+The workloads installed and operated through charms.
+
+Owned by: External entity.
+
+Privileges: As defined by the external entity.
+
+(security-data-flows)=
+## Data flows
+
+This section lists all the main data flows between assets in Juju, along with example threats and the types of controls Juju provides.
+
+(data-flows-admin-user-logging-and-monitoring-systems)=
+### Admin  user - Logging and monitoring systems
+
+Involves the collection, aggregation, and presentation of operational data from the Juju environment. As the Juju environment runs, various components, such as agents (machine / container / unit) and the controller, generate logs and metrics related to system events, performance, and application states. These logs and metrics are collected and stored by the Juju logging and monitoring subsystems.
+
+Example threats:
+- {ref}`threats-availability`: An attacker could target the logging and monitoring system by generating excessive log entries or sending malformed data from compromised Juju agents. This could overwhelm the logging infrastructure, making it slow or unresponsive and resulting in a denial of service (DoS) condition.
+- {ref}`threats-confidentiality`: If logs and monitoring data are transmitted from Juju agents or controllers to a logging and monitoring system without encryption or proper access controls, sensitive information such as usernames, IP addresses, API tokens, or configuration details could be exposed.
+- {ref}`threats-integrity`: If the data flow between Juju administrators and logging/monitoring systems is not protected with integrity checks (e.g., using cryptographic hashes or signatures), an attacker could manipulate the logs or monitoring data in transit.
+
+Controls: [TBA]
```

Comment:
controller-config (`auditing-*`, `max-debug-log-duration`, `model-logfile-*`, `model-logs`, et al)

Reply:
You mean all of these, right?

Controller configurations: {ref}`controller-config-audit-log-capture-args`, {ref}`controller-config-audit-log-exclude-methods`, {ref}`controller-config-audit-log-max-backups`, `controller-config-audit-log-max-size`, {ref}`controller-config-auditing-enabled`, {ref}`controller-config-model-logfile-max-backups`, {ref}`controller-config-model-logfile-max-size`, {ref}`controller-config-model-logs-size`, etc.

Reply:
Yeap.